tests/queries/0_stateless/02668_parse_datetime.sql

```diff
@@ -197,7 +197,7 @@ select parseDateTime(''); -- { serverError NOT_ENOUGH_SPACE }
 
 select parseDateTime64(''); -- { serverError NOT_ENOUGH_SPACE }
 select parseDateTime64('2021-01-04 23:12:34.118'); -- { serverError NOT_ENOUGH_SPACE }
-select parseDateTime64('2177-10-09 10:30:10.123'); -- { serverError CANNOT_PARSE_DATETIME }
+select parseDateTime64('2177-10-09 10:30:10.123'); -- { serverError NOT_ENOUGH_SPACE }
```

Comment:
Why does the same test in the other .sql file (02668_parse_datetime_in_joda_syntax.sql) succeed but here it fails?

Reply:
The error is 
```
Code: 243. DB::Exception: Received from localhost:9000. DB::Exception: Unable to parse fragment %f from 123 because readNumber6 requires size >= 6: In scope SELECT parseDateTime64('2177-10-09 10:30:10.123'). (NOT_ENOUGH_SPACE)
```
In 02668_parse_datetime_in_joda_syntax.sql, `select parseDateTime64InJodaSyntax('2177-10-09 10:30:10.123', 'yyyy-MM-dd HH:mm:ss.SSS');` this works, I think it is the format issue. 
Indeed, we should make this `select parseDateTime64('2177-10-09 10:30:10.123'); ` this SQL succeed, maybe in another PR.